### PR TITLE
Change1

### PR DESCRIPTION
**Pull request by morojenkAM**

### CODE REVIEW

File: src/StreamsAPI_Lambda/Repetitive.java [hunk 1]
```diff
@@ -1,106 +1,21 @@
 package StreamsAPI_Lambda;
 
-import java.util.*;
-import java.util.regex.Matcher;
```
Comment (morojenkAM) [inline src/StreamsAPI_Lambda/Repetitive.java:4]: import Mather ,Pattern you should not have

File: src/StreamsAPI_Lambda/Map/ForEach.java [hunk 2]
```diff
@@ -1,5 +1,18 @@
 package StreamsAPI_Lambda.Map;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Iterator;
```
Comment (morojenkAM) [inline src/StreamsAPI_Lambda/Map/ForEach.java]: import Iterator you should not have

File: src/StreamsAPI_Lambda/List/Stream.java [hunk 3]
```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
         list.add("D");
 
        list.stream().forEach(
-               (temp) -> System.out.println(temp));
+               (listWithElements) -> System.out.println(listWithElements));
```
Comment (olga-casian) [inline src/StreamsAPI_Lambda/List/Stream.java:15]: the argument is not a list, it's a list element

Comment (olga-casian) [inline src/StreamsAPI_Lambda/List/Stream.java:15]:
```suggestion
               (listElement) -> System.out.println(listElement));
```

File: src/StreamsAPI_Lambda/ArrayList/AddAll.java [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import java.util.ArrayList;
```
Comment (olga-casian) [inline src/StreamsAPI_Lambda/ArrayList/AddAll.java]: please read naming conventions - package names section

**Review by olga-casian — APPROVED** [review]
looks good, feel free to merge it!
note for the future: it's usually a good practice to
* add a meaningful PR name, in this case it could be "Applying review comments for Collections and Streams API problems"
* add some description to your PR, it's ok if it's only 1 line; in this case you could link the PRs where the comments were: https://github.com/morojenkAM/Learn_Java_Course/pull/1 and https://github.com/morojenkAM/Learn_Java_Course/pull/2

**Pull request merged**